content/cft/banner.md

```diff
@@ -24,7 +24,8 @@ advanced:
   css_class: 
 ---
 
-Introduction text ...
+THE ACM REP international conference aims to bring together experts and practitioners engaged in the advancement and conduct of reproducible science in computing disciplines. The first-of-its-kind conference will serve as a premier forum for the exchange and presentation of the concepts, tools, techniques, practice and state-of-art in reproducible science. The conference committee invites tutorials related to reproducibility and independent verification. 
 
+For an in-depth list of topics, please see the ACM REP 2023 call for papers.
```

Comment:
I have a commit ready that moves this to topics and includes a link to the history of ACM REP.

Reply:
great, sounds good!